src/ConfigWidgets/SymbolLocationsDialog.cpp

```diff
@@ -106,14 +110,17 @@ SymbolLocationsDialog::~SymbolLocationsDialog() {
 
 SymbolLocationsDialog::SymbolLocationsDialog(
     orbit_client_symbols::PersistentStorageManager* persistent_storage_manager,
-    bool allow_unsafe_symbols, std::optional<const ModuleData*> module, QWidget* parent)
+    orbit_metrics_uploader::MetricsUploader* metrics_uploader, bool allow_unsafe_symbols,
```

Comment:
Nit: Why not using orbit_metrict_uploader::MetricsUploader before?

Reply:
My preference is to only put a `using` directive for things I use multiple times in the the file. for example `orbit_metrics_uploader::ScopedMetric` is used 3 times -> `using`. `orbit_metrics_uploader::MetricsUploader` is only used once -> no using